demo/dlb-init.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+enable_and_configure_vfs() {
+  DEVPATH=$1
```

Comment:
I'd suggest to use lower case names for local variables. It will also fix overlapping names fro global and local DEVPATH and NUM_VFS_PER_PF.

Reply:
locality aspect can be ensured by using keyword `local` which limits the scope:
`local devpath=$1`

Reply:
Aha.! Thanks.! Actually I tried to search about 'naming variables in shell script', but it was not really helpful. Let me change!

Reply:
@okartau Idk why, but 'local' is not found in toybox. I tried to change toybox_config with `make menuconfig`, but does not show anywhere. so, I did not put but just kept them as small letters.

Reply:
@okartau sorry, what do you mean? I already removed `echo | tee`, and `local` is not related to that.!
toybox complained about `echo $num > $file` for some reason, but works with `echo -n $num > $file`.!

Reply:
I mean that toybox is not real bash so we see different effects. I don't know why toybox version works with -n added (while bash version works without -n), but I guess thats alright, we don't need ending newline in these entries. We can forget about `local` in this context then. So lets resolve  this thread.